src/rider/main/kotlin/com/jetbrains/rider/plugins/efcore/features/dbcontext/scaffold/ScaffoldDbContextDialogWrapper.kt

```diff
@@ -262,9 +262,12 @@ class ScaffoldDbContextDialogWrapper(
             tablePanel.isVisible = !enabledCheckbox!!.isSelected
 
             row {
+                val scaffoldAllTables = enabledCheckbox!!.selected
```

Comment:
I have concerns about naming here - `createToggleableTablePanel` function is used for both "Tables" and "Schemas" tabs. I propose to use such naming:
- `scaffoldAllTables` → `scaffoldAll`: boolean representing if all tables or all schemas (**but not both**) should be scaffolded
- `tablesList` → `scaffoldItems`: list representing whitelist for tables or for schemas (depending on the current tab)

Feel free to propose other naming resolution if you don't like that one.

```suggestion
                val scaffoldAll = enabledCheckbox!!.selected
```

Reply:
You made me realize a bug that was present aswell :D.
On latest commit I renamed it properly and made it support both schemas and tables